src/asset_folder_importer/premiere_get_referenced_media/processor.py

```diff
@@ -140,6 +140,17 @@ def process_premiere_fileref(filepath, server_path, vsproject, vs_pathmap=None,
     return item
 
 
+def update_metadata(vsproject, invalid_media_paths):
```

Comment:
pls update this function name to a better description before deployment

Reply:
Will do.